README.md

```diff
@@ -1,15 +1,17 @@
 # <img src="https://raw.githubusercontent.com/flybywiresim/fbw-branding/master/svg/FBW-Logo.svg" placeholder="FlyByWire" width="400"/>
 
 
-[![Discord](https://img.shields.io/discord/738864299392630914.svg?label=&logo=discord&logoColor=ffffff&color=7389D8&labelColor=6A7EC2)](https://discord.gg/UjzuHMU)
+[![Discord](https://img.shields.io/discord/738864299392630914.svg?label=&logo=discord&logoColor=ffffff&color=7389D8&labelColor=6A7EC2)](https://discord.gg/UjzuHMU) 
+
+[![Twitter](https://img.shields.io/twitter/follow/FlybywireSim?style=flat&logo=twitter)](https://twitter.com/FlybywireSim)

```

Comment:
Since you added the Twitter, you might as well add our YouTube/Facebook accounts